### PR TITLE
Updating GitHub links to be used consistently in Federalist application

### PR DESCRIPTION
**Pull request by line47**
Addresses issue https://github.com/18F/federalist/issues/1334 and https://github.com/18F/federalist/issues/1266

### Screenshots 
screen shot 2017-12-21 at 4 40 54 pm
screen shot 2017-12-21 at 4 40 44 pm
screen shot 2017-12-21 at 4 40 31 pm
screen shot 2017-12-21 at 4 40 16 pm

### CODE REVIEW

**Comment by line47** [comment]
@jseppi or @el-mapache you all might have better luck with the tests that I have 🤞 🍀

**Comment by jseppi** [comment]
@el-mapache: for context, we're running into the issue of enzyme apparently not being able to render or otherwise pull anything out from inside of the `GitHubLink` component. Ezyme always renders those components as weirdly self-closed `<wrapper />` elements.

**Comment by jseppi** [comment]
I think I can get this working via `.html()`

**Comment by jseppi** [comment]
Tests should be good now. Would be good to get @el-mapache's eyes on this, particularly my commit https://github.com/18F/federalist/pull/1401/commits/bc9339f24f1c71d201b68f18f1c0d85d684eb0c6

**Comment by line47** [comment]
Thanks @jseppi and @el-mapache - good to merge this now? 🚀

File: frontend/sass/_header.scss [hunk 1]
```diff
@@ -81,7 +81,7 @@ $logo-height: 29px;
 }
 
 .header .usa-width-two-thirds {
-  display: flex;
+  // display: flex;
```
Comment (jseppi) [inline frontend/sass/_header.scss]: should we just get rid of the entire selector?

Comment (line47) [reply frontend/sass/_header.scss]: Yeah I think using the `usa-width` classes OR building our own flex is the way to go. Using both gets some strange results.

**Comment by line47** [comment]
@jseppi I couldn't find the underline comment

**Comment by jseppi** [comment]
Maybe I didn't actually make my comment about the dangling underline. I can't find it either o_O.

Is there any way to get rid the dangling underlines between the link text and GitHub icons? Like in these:

screen shot 2017-12-22 at 9 00 25 am

screen shot 2017-12-22 at 9 00 20 am

**Comment by line47** [comment]
Yeah let me take a peek 👀

**Comment by line47** [comment]
Fixed the links 👍 
screen shot 2017-12-22 at 10 19 25 am
screen shot 2017-12-22 at 10 19 32 am

**Comment by jseppi** [comment]
I'm happy to fix the failing test if you'd like, @line47

**Comment by line47** [comment]
I think I fixed the test 👍  pushing back up now

**Pull request merged**